src/DotNetWorker.Core/Converters/Converter/EnableConvertersFallbackAttribute.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Azure.Functions.Worker.Converters
+{
+    /// <summary>
+    /// An attribute that specifies if Converters fallback is enabled
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Class)]
+    public sealed class EnableConvertersFallbackAttribute : Attribute
```

Comment:
This needs a bool parameter so that it can be explicitly enabled _or_ disabled.

Reply:
Added